cg/cli/backup.py

```diff
@@ -18,25 +18,30 @@ def backup(context: click.Context):
 
 @backup.command("fetch-flowcell")
 @click.option("-f", "--flowcell", help="Retreive a specific flowcell")
-@click.option("--dry", is_flag=True, help="Only log, do not contact PDC")
+@click.option(
+    "--dry-run", "dry_run", is_flag=True, help="Don't retrieve from PDC or set flowcell's status"
```

Comment:
```suggestion
    "--dry-run", is_flag=True, help="Don't retrieve from PDC or set flowcell's status"
```
This is not needed here so I removed it